.github/workflows/commit-format.yml

```diff
@@ -91,6 +91,23 @@ jobs:
             fi
           done;
 
+          ## check no Signed-of-by in the middle.
```

Comment:
```suggestion
          ## check no Signed-off-by in the middle.
```

Reply:
I think the gcc-commit-mklog is not handling this because the SoB is still rarely used in GCC, so the intersection with dev using this script is even smaller (if not empty... until gccrs?). But good point, I'll hack it to have it put the line at the end.